lib/div-alg.gi

```diff
@@ -1963,7 +1963,7 @@ for i in [1..k] do
         d := Gcd(m,r1-1);
         md := m/d;
         a := Int(ZmodnZObj(e[i,j]/d,md)*ZmodnZObj((r1-1)/d,md)^-1 );
-        Add(as,List([0..d-1],y->a+y*md mod m));
+        Add(as,List([0..d-1],y->(a+y*md) mod m));
```

Comment:
Commit message "Update div-alg.gi" is redundant. It can be seen from the commit which file has been updated. I would formulate it as "Fix incorrect reduction mod m due to missing parentheses".